app/views/pages/home.html.erb

```diff
@@ -1,2 +1,3 @@
 <h1>Welcome to my awesome website !</h1>
+<h2>It's <span style="color: red;">amazing!!!</span></h2>
```

Comment:
As we saw together we don't want the amazing word in our homepage. Can you change it ?

Reply:
Awesome please

Reply:
`<h2>It's <span style="color: red;">awesome!!!</span></h2>`

Reply:
@nadiaauger fixed!